lldb/source/Plugins/ObjectFile/Minidump/MinidumpFileBuilder.cpp

```diff
@@ -126,6 +129,17 @@ Status MinidumpFileBuilder::AddDirectory(StreamType type,
   return error;
 }
 
+Status MinidumpFileBuilder::AddLLDBGeneratedStream() {
+  Status error;
+  StreamType type = StreamType::LLDBGenerated;
```

Comment:
```suggestion
  llvm::support::ulittle32_t type = StreamType::LLDBGenerated;
```
.. for the same reason. (You may need to add some static_casts to get the initialization to compile)

Reply:
Dropped the magic bytes and added empty stream. Resolving